modules.d/01systemd-pcrphase/module-setup.sh

```diff
@@ -26,8 +30,11 @@ depends() {
 # Install the required file(s) and directories for the module in the initramfs.
 install() {
 
+    binary="$systemdutildir/systemd-pcrphase"
```

Comment:
You can just put both binaries to the `inst_multiple` call. It uses `-o` (optional), hence non-existing files are ignored.

Reply:
Is there any better form for that `require_binaries` call?  as to the `inst_multiple`- and I'm probably being anal here- but that install should probably only allow optional for the `*.conf`.  The `*.service` definitely need to exist for this module to work.

Where I'm going with that is thus: how do I make this *fail* if one of those two binaries isn't found?  That problem can only occur if there is a time-of-check-time-of-use, but it's probably best to note the "I'm probably being anal here" comment from above.

Reply:
> The `*.service` definitely need to exist for this module to work.

Well, you could change this to use `inst_multiple` for stuff that *must* be present and `inst_multiple -o` for stuff that's *may* be present. But IMHO it's not strictly needed. `dracut` assumes that the stuff that's supposed to work in the initrd being created does work in the "source" system, which couldn't be true if some required files were missing.
 
> Where I'm going with that is thus: how do I make this _fail_ if one of those two binaries isn't found? 

You already do, in `check()`. Nothing else is needed.

Reply:
Pushing a rewrite w/ the requested changes; this should be good to go w/ what you've asked for.  I'm not splitting up the inst_multiple for the reasons you mentioned, and lazyness on my part. ;)